packages/react-native-drawer-layout/src/views/Drawer.tsx

```diff
@@ -9,10 +9,10 @@ import {
   useWindowDimensions,
   View,
 } from 'react-native';
+import { Gesture } from 'react-native-gesture-handler';
```

Comment:
Can't import anything from `react-native-gesture-handler` here. Move it to `GestureHandlerNative.tsx`.